collectors/VMStatsCollector.py

```diff
@@ -0,0 +1,47 @@
+
+from BaseCollector import BaseCollector
+import os, time
+import requests
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class VMStatsCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.statkey_yaml = YamlRead('collectors/statkey.yaml').run()
+        self.g = GaugeMetricFamily('vrops_vms_stats', 'testtext', labels=['cluster', 'datacenter', 'virtualmachine', 'hostsystem', 'statkey'])
```

Comment:
please rename cluster to: `vccluster`
cluster is predefined and shouldn't be used

Reply:
This has been taken care and cluster is renamed to vccluster.